ytdl_nfo/nfo.py

```diff
@@ -85,7 +85,7 @@ def print_nfo(self):
 
     def write_nfo(self, filename):
         xmlstr = minidom.parseString(ET.tostring(self.top, 'utf-8')).toprettyxml(indent="    ")
-        with open(filename, 'w') as f:
+        with open(filename, 'wt', encoding="utf-8") as f:
```

Comment:
Otherwise, non ASCII text in video titles and descriptions can cause an exception to be raised here